cmd/lotus-storage-miner/info.go

```diff
@@ -37,6 +37,13 @@ var infoCmd = &cli.Command{
 
 		fmt.Printf("Miner: %s\n", maddr)
 
+		// Sector size
+		size, err := api.StateMinerSectorSize(ctx, maddr, nil)
+		if err != nil {
+			return err
+		}
+		fmt.Printf("Sector Size(Byte): %d\n", types.NewInt(size))
```

Comment:
I'd go with:
```suggestion
		fmt.Printf("Sector Size: %dKiB\n", size / 1024)
```

No need to convert to BigInt, and as I don't think we'll ever have sectors smaller than 1K, even for testing, displaying the size in kilobytes is much more readable

Reply:
maybe I can format it to KiB/MiB/GiB. wait for a moment.